packages/prisma/schema.prisma

```diff
@@ -708,6 +708,8 @@ model Availability {
 }
 
 model SelectedCalendar {
+  // Will become required and @id in a seperate PR
+  id                               String?               @default(uuid())
```

Comment:
How about making it @unique as well so that we don't have accidental same UUID through custom migration

Reply:
@unique is not supported on nullable field :(

Reply:
Manually added to the migration.